book/src/composite_templates.md

```diff
@@ -45,25 +45,23 @@ Filename: <a class=file-link href="https://github.com/gtk-rs/gtk4-rs/blob/master
 ```
 
 Now we have to compile the resources and link it to our application.
-One way to do this is to execute [`gio::compile_resources`](https://gtk-rs.org/gtk-rs-core/stable/latest/docs/gio/fn.compile_resources.html) within a cargo [build script](https://doc.rust-lang.org/cargo/reference/build-scripts.html).
+One way to do this is to execute [`gio::compile_resources`](https://gtk-rs.org/gtk-rs-core/stable/latest/docs/glib-build-tools/fn.compile_resources.html) within a cargo [build script](https://doc.rust-lang.org/cargo/reference/build-scripts.html).
```

Comment:
This link is going to be broken until docs are built for glib-build-tools, is that fine?